ft/docker/docker-compose.yml

```diff
@@ -14,6 +14,6 @@ services:
       - ./employee-service.yaml:/application.yaml
 
   mockserver:
-    image: ${DOCKER_REGISTRY}/mockserver/mockserver:mockserver-5.11.1
+    image: docker.io/mockserver/mockserver:mockserver-5.11.1
```

Comment:
Why not to use variable instead of hardcoded URI

Reply:
I propose to introduce variable for docker.io in order have possibility  to use proxy

Reply:
FIXED